test/elixir-mode-font-test.el

```diff
@@ -462,12 +463,10 @@ end
    (should (eq (elixir-test-face-at 32) 'elixir-ignored-var-face))
    (should (eq (elixir-test-face-at 38) 'elixir-ignored-var-face))
    (should (eq (elixir-test-face-at 40) 'elixir-ignored-var-face))
-   (should (eq (elixir-test-face-at 46) 'font-lock-constant-face))
-   (should (eq (elixir-test-face-at 46) 'font-lock-constant-face))
-   (should (eq (elixir-test-face-at 52) 'font-lock-constant-face))
-   (should (eq (elixir-test-face-at 53) 'font-lock-constant-face))
-   (should (eq (elixir-test-face-at 55) 'font-lock-constant-face))))
+   (should (eq (elixir-test-face-at 52) 'elixir-atom-face))
```

Comment:
Can you add a comment above here that says what characters are at the positions being tested? I know it's not done anywhere else but it makes reviewing these much easier.